dio/lib/src/adapter.dart

```diff
@@ -4,7 +4,7 @@ import 'dart:typed_data';
 import 'package:meta/meta.dart';
 
 import 'adapters/io_adapter.dart'
-    if (dart.library.html) 'adapters/browser_adapter.dart' as adapter;
```

Comment:
Does this require a certain Dart version?

Reply:
Tested locally (with 3.16) and it seems working